### ISSUE TITLE
Toast: element missing alert role to announce contents (4.1.2, 3.3.1)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#871]
### Expected behavior

Toasts are alerts which are read to users who are using assistive technology. Alert element has role="alert" and aria-live="polite" so its contents will be read out at next convenient point for someone listening. Feature teams can override aria-live with "assertive" if they have an error message which is of importance to interrupt the user with. 

https://www.w3.org/TR/wai-aria-practices/#alert
https://www.w3.org/TR/wai-aria-1.1/#alert
### Actual behavior

Element doesn't have role and would be difficult to discover for those using assistive technologies.
